### PR TITLE
Make terraform 0.12 the primary terraform installed

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
This swaps tf11 and tf12 so that we are not using tf12 by default in the `latest` image. I keep the `tf12` tag around for compatibility but honestly we could remove it.

### CODE REVIEW

**Review by esacteksab — APPROVED** [review]

**Review by esacteksab — CHANGES_REQUESTED** [review]
We actually have a ticket in our backlog to do this, so thanks for picking this up. It calls out `gawk` and if you see the TF 12 Docker file, there is something there to install `gawk` that isn't there in the default Dockerfile https://github.com/trussworks/circleci-docker-primary/blob/cca6ecc6b3be8b1c153f292e9ec5e57faf250be5/tf11/Dockerfile#L18

**Comment by chrisgilmerproj** [comment]
@esacteksab I've moved the install of `gawk` over to the main file. Nice catch.

**Review by esacteksab — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -72,13 +72,13 @@ RUN set -ex && cd ~ \
   && : Install Node 10.x \
   && curl -sS https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
   && echo "deb https://deb.nodesource.com/node_10.x $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/nodesource.list \
-  && apt-get -qq update \
```
Comment (brainsik) [inline Dockerfile:75]: @chrisgilmerproj Why did these `update` commands get removed? It used to be the case that `apt-get` wouldn't see the packages from the newly added apt source unless that got ran.

File: Dockerfile [hunk 2]
```diff
@@ -72,13 +72,13 @@ RUN set -ex && cd ~ \
...
   && apt-get -qq -y install --no-install-recommends nodejs \
   && : Install Yarn \
   && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
   && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
-  && apt-get -qq update \
```
Comment (brainsik) [inline Dockerfile:80]: See above comment.

File: Dockerfile (hunk 1 above)
Comment (chrisgilmerproj) [reply Dockerfile:75]: I think I see the problem, let me make a change to our dockerfile and push a new update.

Comment (chrisgilmerproj) [reply Dockerfile:75]: See #84 for updates

**Pull request merged**